Source/Csla.Blazor/CslaPermissionsPolicyProvider.cs

```diff
@@ -16,30 +18,32 @@ namespace Csla.Blazor
   /// </summary>
   public class CslaPermissionsPolicyProvider : IAuthorizationPolicyProvider
   {
-    /// <summary>
-    /// Gets the fallback policy provider
-    /// </summary>
-    public DefaultAuthorizationPolicyProvider FallbackPolicyProvider { get; }
+    private readonly AuthorizationOptions _options;
 
     /// <summary>
     /// Creates an instance of the type
     /// </summary>
     /// <param name="options">Authorization options</param>
     public CslaPermissionsPolicyProvider(IOptions<AuthorizationOptions> options)
     {
-      FallbackPolicyProvider = new DefaultAuthorizationPolicyProvider(options);
+      if (options == null)
+      {
+        throw new ArgumentNullException(nameof(options));
```

Comment:
Do we want an exception? Or to get the fallback from a `DefaultAuthorizationPolicyProvider`?

Reply:
`DefaultAuthorizationPolicyProvider` is already added as part of the ASPNET Host Defaults. I'm not sure that it's valid for options to be null. 

Incidentally when do CslaPolicy's get used? I haven't see any policy names starting with 'Csla:' even though I have `AuthorizationRules`

Reply:
That makes sense about the options.

Can you ask about the CslaPolicy stuff in a new Discussion? This isn't a great location for that conversation.